NeuralABC_tools/EigenGame_v1.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+
+def EigenGame2(data, n):
+    M = np.matmul(data.T, data)
+    dim = M.shape[0]
+    vecs = []
+    gradients = []
+    
+    step = 0.01
+    
+    for i in range(n):
+        vecs.append(np.random.rand(dim))
```

Comment:
I think it is more efficient to write this as one function rather than a loop. define vecs as a random floats with shape ```nXfeats```.